### PR TITLE
Squashing a few compiler warnings.

### PR DESCRIPTION
**Pull request by Adenilson**

### CODE REVIEW

**Comment by metajack** [comment]
r+

File: components/devtools/actors/timeline.rs [hunk 1]
```diff
@@ -214,7 +214,7 @@ impl Actor for TimelineActor {
                 **self.is_recording.lock().as_mut().unwrap() = true;
 
                 let (tx, rx) = channel::<TimelineMarker>();
-                self.script_sender.send(SetTimelineMarkers(self.pipeline, self.marker_types.clone(), tx));
+                let _ = self.script_sender.send(SetTimelineMarkers(self.pipeline, self.marker_types.clone(), tx));
```
Comment (SimonSapin) [inline components/devtools/actors/timeline.rs]: Unless there is a specific reason to ignore errors, use `.unwrap()` on the `Result` instead.

File: components/devtools/actors/timeline.rs [hunk 2]
```diff
@@ -254,7 +254,7 @@ impl Actor for TimelineActor {
                 };
 
                 stream.write_json_packet(&msg);
-                self.script_sender.send(DropTimelineMarkers(self.pipeline, self.marker_types.clone()));
+                let _ = self.script_sender.send(DropTimelineMarkers(self.pipeline, self.marker_types.clone()));
```
Comment (SimonSapin) [inline components/devtools/actors/timeline.rs]: Same here.

File: components/script/dom/window.rs [hunk 3]
```diff
@@ -805,7 +805,7 @@ impl<'a> WindowHelpers for JSRef<'a, Window> {
     fn emit_timeline_marker(self, marker: TimelineMarker) {
         let sender = self.devtools_marker_sender.borrow();
         let sender = sender.as_ref().expect("There is no marker sender");
-        sender.send(marker);
+        let _ = sender.send(marker);
```
Comment (SimonSapin) [inline components/script/dom/window.rs]: And here.

File: components/script/script_task.rs [hunk 4]
```diff
@@ -1349,7 +1349,7 @@ impl ScriptTask {
     fn emit_timeline_marker(&self, marker: TimelineMarker) {
         let sender = self.devtools_marker_sender.borrow();
         let sender = sender.as_ref().expect("There is no marker sender");
-        sender.send(marker);
+        let _ = sender.send(marker);
```
Comment (SimonSapin) [inline components/script/script_task.rs]: And here.

**Comment by metajack** [comment]
@bors-servo: r-

Based on Simon's feedback.

**Comment by nox** [comment]
Would it be too overwhelming to use Option::expect() in many more places to give better messages when panicking?

**Comment by Adenilson** [comment]
Should be ok now.

**Comment by SimonSapin** [comment]
@nox, it’s not always easy to come up with a message for `expect` that will be more helpful than the filename and line number that `RUST_BACKTRACE=1` can already provide. It wouln’t be an error message meant for users: `unwrap` or `expect` should only be used when failing indicates a bug.

**Pull request merged**